cmd/main.go

```diff
@@ -0,0 +1,46 @@
+package main
```

Comment:
(nit) main.go could go at the root of the project for now. Putting it under ./cmd/.../. is useful when your project produces more than 1 binary.